pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -204,27 +268,8 @@ func findMinMTU() (*net.Interface, error) {
 	return &intfs[defIntfIndex], nil
 }
 
-const NET_CONFIG_TEMPLATE = `{
```

Comment:
Any reason this got moved to the top? Golang doesn't care where it lives so this is superfluous diffage.

Reply:
because it is a const, and because it is annoying to scan the entire code file to find something that should be defined early on. We have practically visited every part of the code file, having one more change won't hurt.